api/uniter/action_test.go

```diff
@@ -46,7 +46,10 @@ func (s *actionSuite) TestAction(c *gc.C) {
 
 	for i, actionTest := range actionTests {
 		c.Logf("test %d: %s", i, actionTest.description)
+		operationId, err := s.Model.EnqueueOperation("a test")
```

Comment:
go idiomatic gripe of mine, should be operationID..

Reply:
righto